BUILD

```diff
@@ -13,3 +13,5 @@
 # limitations under the License.
 
 exports_files(["LICENSE"])
+
+exports_files(["tsconfig.json"], visibility = ["//visibility:public"])
```

Comment:
Nit: trailing newline (also a couple other files)

Reply:
Do you know if there's a way to enforce this? (Or is enforcing even the way to go)

Reply:
(Done)

Reply:
I don't know of one, unfortunately.